open-company/open-company-web
# Integrate Sentry with desktop app

Trello: https://trello.com/c/MG1fUJZF/2140-wrap-electron-and-mobile-shells-in-sentry-to-capture-exceptions-crashes

### To test

- Add a call to an undefined function [right here](https://github.com/open-company/open-company-web/blob/be04c898826cb1c8c5ddd36dbf42c4ce25e0db0a/src/oc/electron/main.cljs#L13) in the electron main process

For example:
```
(my-undefined-function)
```

- Build the desktop application using `boot staging-electron`. You **must** use either staging or production to test this. Sentry is not configured for development builds.
- Run the desktop app (`cd target`, `yarn install && yarn start`)
- [x] Do you see the caught exception [in Sentry](https://sentry.io/organizations/opencompany/issues/?project=1509179)
- Remove all local changes to the code
